gcp_variant_transforms/libs/bigquery_vcf_schema.py

```diff
@@ -258,17 +264,26 @@ def get_rows_from_variant(variant, split_alternate_allele_info_fields=True):
 
 
 def _get_call_record(call):
-  """A helper method for ``get_rows_from_variant`` to get a call as JSON."""
+  """A helper method for ``get_rows_from_variant`` to get a call as JSON.
+
+  Args:
+    call: PyVCF Variant call to convert.
+
+  Returns:
+    BigQuery call value (dict).
+  """
   call_record = {
       ColumnKeyConstants.CALLS_NAME: _get_bigquery_sanitized_field(call.name),
       ColumnKeyConstants.CALLS_PHASESET: call.phaseset,
       ColumnKeyConstants.CALLS_GENOTYPE: call.genotype or []
   }
+  is_empty = _is_empty_field('genotype', call.genotype)
```

Comment:
What do you think of treating genotype as a special field and just using is_empty_field for other info fields? This avoids special-casing genotype inside is_empty_field
In other words, something like: `is_empty = set(call.genotype) == set((vcfio.MISSING_GENOTYPE_VALUE,))`

Reply:
sure! done. i actually considered this originally, but didn't do it because the boolean value check gets duplicated. i'm fine either way though!